docs/konnectors.md

```diff
@@ -320,18 +320,40 @@ On the host create the file /etc/rkt/net.d/10-containers.conf
 
 and run your container with the "--net=bridge" option. That way, a new interface is available in the container and gives you access to the host.
 
+## Konnector install and run details
+
+### Install
+
+The konnectors will be installed in the .cozy_konnectors directory (not in VFS ?) using npm install
```

Comment:
I would advise against using `npm` (or `yarn`) to install konnectors on production. During development it may be useful to use such dependency managements systems, however I think they would cause issues on our systems in production:

  - they make tons of I/O requests and are usually resource intensive
  - they do not provide a real reproducible build (even if `yarn` seems to be a little better than `npm` on this regard)
  - they do not provide enough security regarding the code that is actually downloaded (leftpadgate) (even if yarn is a also little better in this regard with the `yarn.lock` file)
  - requires a new dependency on quite complex programs, which mean we have to package the `cozy-stack` with a specific version `npm` or `yarn` if we want to stay dep free
  - we cannot provide a simple signing mechanism for our contributors to sign and verify the integrity of their applications
  - we are more coupled with the javascript ecosystem and it will be more difficult to allow a konnector to be written in another language in the future

I would argue that konnectors contributors should provide a versioned and packaged archive containing all the code to run their application.

However, I agree that without such tools, we have to develop a new hosting and versioning solution. Maybe our marketplace should provide these possibilities.